thanos/templates/store-deployment.yaml

```diff
@@ -117,8 +117,12 @@ spec:
       volumes:
       - name: data
       {{- if $root.Values.store.dataVolume.backend }}
+        {{- if $root.store.persistentVolumeClaim }}
```

Comment:
@tarokkk @ahma this change actually broke the PVC usage. 

it should be `$root.values.store.persistentVolumeClaim `